public/javascripts/main.js

```diff
@@ -41,7 +41,47 @@ David end
 Michael start
 */
 
-
+//function to close chat
+closechat.onclick = function (){
```

Comment:
FYI, the best practice might be using document.getElementById instead of using the id of the element directly.

This is the reference:
https://stackoverflow.com/questions/3434278/do-dom-tree-elements-with-ids-become-global-variables/3434388#3434388